### ISSUE TITLE
Add calculation of integrated luminosity via lumiCalc2 script

### ISSUE THREAD

**Issue by vkuznet** [dmwm/cmssh#33]

**Comment by vkuznet** [comment]
I need to install coral and checkout from CVS RecoLuminosity/LumiDB. Then I can add into info command for dataset the parallel look-up of its lumi numbers.

**Comment by vkuznet** [comment]
Once coral is installed, I need to do several actions
1. Link liblcg_PyCoral.dylib to liblcg_PyCoral.so, liblcg_ConnectionService.dylib to liblcg_ConnectionService.so on OSX
2. create soft/CMSSW/SITECONF/local/JobConfig/site-local-config.xml where content of site-local-config.xml I can look at /afs/cern.ch/cms/SITECONF/local/JobConfig/site-local-config.xml
This file itself contains reference to /afs/cern.ch/cms/SITECONF/local/PhEDEx/storage.xml, therefore it would be wise if I'll put them somehow on local disk.
3. I need to setup environment to run lumiCalc script

```
ver=CORAL_2_3_21-cms20
coral=$CMSSH_ROOT/CMSSW/$SCRAM_ARCH/cms/coral/$ver/$SCRAM_ARCH
coral_external=$CMSSH_ROOT/CMSSW/$SCRAM_ARCH/cms/coral/$ver/external/$SCRAM_ARCH/lib
export PYTHONPATH=$PYTHONPATH:$PWD:$coral/python:$coral/lib:$coral_external
export PATH=$PATH:$PWD/tmp/RecoLuminosity/LumiDB/scripts
export DYLD_LIBRARY_PATH=$DYLD_LIBRARY_PATH:$coral/lib:$coral_external
export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$coral/lib:$coral_external
python $PWD/tmp/RecoLuminosity/LumiDB/scripts/lumiCalc2.py -i collision.json --without-checkforupdate --without-correction delivered
```

please note that I need to run lumiCalc script without-checkforupdate since it will ask for CVS access to repository. This should be set in setup.sh

**Comment by vkuznet** [comment]
Done and released in v0.25
